specification/storage/resource-manager/Microsoft.Storage/stable/2025-01-01/storage.json

```diff
@@ -3694,6 +3750,16 @@
       },
       "description": "Routing preference defines the type of network, either microsoft or internet routing to be used to deliver the user data, the default option is microsoft routing"
     },
+    "DualStackEndpointPreference": {
```

Comment:
Please add swagger example to cover the new properties change.

Reply:
added